Dependencies/CMakeLists.txt

```diff
@@ -64,16 +57,6 @@ endif()
 
 # -------------------------------- glslang --------------------------------
 # Dependencies: none
-set(BUILD_EXTERNAL OFF CACHE BOOL "")
-set(ENABLE_SPVREMAPPER OFF CACHE BOOL "")
-set(ENABLE_GLSLANG_BINARIES OFF CACHE BOOL "")
-set(ENABLE_HLSL OFF CACHE BOOL "")
-set(ENABLE_OPT OFF CACHE BOOL "")
-set(ENABLE_CTEST OFF CACHE BOOL "")
-set(SKIP_GLSLANG_INSTALL ON CACHE BOOL "")
-set(ENABLE_GLSLANG_WEBMIN ON CACHE BOOL "")
-set(ENABLE_GLSLANG_WEBMIN_DEVEL OFF CACHE BOOL "")
-add_subdirectory(glslang)
 set_property(TARGET GenericCodeGen PROPERTY FOLDER Dependencies/glslang)
 set_property(TARGET glslang PROPERTY FOLDER Dependencies/glslang)
 set_property(TARGET OGLCompiler PROPERTY FOLDER Dependencies/glslang)
```

Comment:
Do you meqn moving these set_property to root cmakelists?

Reply:
I don't know if we should put everything in root cmakelists or put it in a separate file, but these should go with the other lines that deal with these dependencies so that they are together?